src/components/ChatEntry.js

```diff
@@ -1,22 +1,34 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.js';
 
-const ChatEntry = (props) => {
+const ChatEntry = ({ sender, body, timeStamp, updateLikes }) => {
+  let senderClass = 'chat-entry local';
+  if (sender === 'Estragon') {
+    senderClass = 'chat-entry remote';
+  }
```

Comment:
Great work differentiating the chat participants using props and a variable to change the class!